server/server/app.py

```diff
@@ -0,0 +1,57 @@
+from flask import Flask, redirect, url_for
+import json
+from server.server.Utils import Connection, and_cond
+from server.server.models import Products, CartItem, User
+import flask_login
+app = Flask(__name__)
+
+conn = Connection(local=False)
+
+
+def extract_dict_from_query(*query):
```

Comment:
Seems like your method return a list but is called `extract_dict_from_query` - this is confusing, please change